libs/hardhat/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "@zk-groups/hardhat",
-    "type": "module",
     "version": "0.1.1",
+    "main": "src/index.ts",
```

Comment:
@saleel This should automatically be added by NX.

It generates:

```json
  "main": "./src/index.js",
  "typings": "./src/index.d.ts"
```

Is it correct?

Reply:
But we dont have a `index.js` file in src 🤔